src/components/MonthCalendar.spec.tsx

```diff
@@ -169,4 +169,52 @@ describe('MonthCalendar', () => {
             });
         });
     });
+
+    it('shows back and forward widget buttons', () => {
+        mountWithProviders(
+            <MonthCalendar events={ dummyEvents } focusDate={ new Date(2021, 4, 10) } onFocusDate={ () => null }  />,
+        );
+        cy.get('[data-testid="back-button"]').should('be.visible');
+        cy.get('[data-testid="fwd-button"]').should('be.visible');
+    });
+
+    it('shows the current displayed month in the widget', () => {
+        mountWithProviders(
+            <MonthCalendar events={ dummyEvents } focusDate={ new Date(2021, 4, 10) } onFocusDate={ () => null }  />,
+        );
+        cy.get('[data-testid="selected-month"]').contains('May');
+        cy.get('[data-testid="selected-month"]').contains('2021');
+    });
+
+    it('sets the focus date a month ago when back is clicked', () => {
+        const spyOnFocusDate = cy.spy();
+        mountWithProviders(
+            <MonthCalendar events={ dummyEvents } focusDate={ new Date(2021, 4, 10) } onFocusDate={ spyOnFocusDate }  />,
+        );
+
+        cy.findByText('misc.calendar.prev')
+            .click({ force: true })
+            .then(() => {
+                const date = new Date(2021, 3, 10);
+                expect(spyOnFocusDate).to.be.calledOnce;
+                expect(spyOnFocusDate.args[0][0] instanceof Date).to.be.true;
```

Comment:
Did you try using [`to.be.an.instanceof`](https://www.chaijs.com/api/bdd/#method_instanceof)?